circle.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
```

Comment:
Hmm, this is probably something you won't want to upstream.  What do think is the best way to handle this on our end @rickyhan?  We could probably just keep it on our fork; there shouldn't be any conflicts when rebasing if all of the changes are centralized to this file.